front/src/applications/stdcm/components/StdcmForm/StdcmOperationalPoint.tsx

```diff
@@ -24,12 +24,15 @@ function formatChCode(chCode: string) {
   return chCode === '' ? 'BV' : chCode;
 }
 
-const StdcmOperationalPoint = ({ point, opPointId, disabled }: StdcmOperationalPointProps) => {
+const StdcmOperationalPoint = ({ location, pathStepId, disabled }: StdcmOperationalPointProps) => {
   const { t } = useTranslation('stdcm');
   const dispatch = useAppDispatch();
 
   const { searchTerm, chCodeFilter, sortedSearchResults, setSearchTerm, setChCodeFilter } =
-    useSearchOperationalPoint({ initialSearchTerm: point.name, initialChCodeFilter: point.ch });
+    useSearchOperationalPoint({
+      initialSearchTerm: location?.name,
+      initialChCodeFilter: location?.secondary_code || undefined,
```

Comment:
This should also work I think

```suggestion
      initialChCodeFilter: location?.secondary_code,
```

Reply:
No it does not work because secondary_code can be null or ''

Reply:
Oups the comments aren't in the good order. This one was related to the one below :)

Reply:
Ah yes, now you're right :)